src/draw/lv_draw_vector_private.h

```diff
@@ -70,30 +77,53 @@ struct _lv_vector_stroke_dsc_t {
     lv_matrix_t matrix;
 };
 
-struct _lv_vector_draw_dsc_t {
+/**
+ * Stores how to fill, stroke, transform etc a given path
+ */
+struct _lv_vector_path_attr_t {
     lv_vector_fill_dsc_t fill_dsc;
     lv_vector_stroke_dsc_t stroke_dsc;
     lv_matrix_t matrix;
     lv_vector_blend_t blend_mode;
     lv_area_t scissor_area;
 };
 
-struct _lv_draw_vector_task_dsc_t {
+struct _lv_draw_vector_dsc_t {
     lv_draw_dsc_base_t base;
-    lv_ll_t * task_list; /*draw task list.*/
-};
 
-struct _lv_vector_dsc_t {
-    lv_layer_t * layer;
-    lv_vector_draw_dsc_t current_dsc;
-    /* private data */
-    lv_draw_vector_task_dsc_t tasks;
+    lv_vector_path_attr_t current_dsc;
```

Comment:
`draw_task` should not contain `current_dsc`, which is a structure only needed for drawing and it does not make sense to include it here. Because when the vector drawing task is sent to the backend, only `task_list` is necessary.